packages/cli/src/push/abstract.push.ts

```diff
@@ -16,16 +17,20 @@ export interface AbstractPushEvents {
  *
  * @emits message when a message is received from a client
  */
+@Service()
```

Comment:
Should we decorate abstract classes as `Service`? They shouldn't be instantiable right?

Reply:
no, but the constructor is reused by the derived classes. this also seems to be the recommended way in typedi.